api/src/main/java/org/openmrs/scheduler/Schedule.java

```diff
@@ -231,21 +226,6 @@ public void setRepeatInterval(long repeatInterval) {
 	}
 	
 	/**
-	 * Get the date format used to set the start time.
-	 */
-	public String getDateFormat() {
```

Comment:
@Nuwantha I think its best you exclude these for now from your PR. I am unsure if we should simply remove them.